commonutils.py

```diff
@@ -163,10 +163,10 @@ def setup_nrnbbcore_register_mapping(rings):
                         recordlist.append(v)
                     isec += 1
 
-        #register soma section list
-        pc.nrnbbcore_register_mapping(gid, "soma", somasec, somaseg)
+                #register soma section list
+                pc.nrnbbcore_register_mapping(gid, "soma", somasec, somaseg)
 
-        #register dend section list
-        pc.nrnbbcore_register_mapping(gid, "dend", densec, denseg)
+                #register dend section list
+                pc.nrnbbcore_register_mapping(gid, "dend", densec, denseg)
 
```

Comment:
@nrnhines : if I am not mistaken, calls to `pc.nrnbbcore_register_mapping` are outside loop over gid on rank and hence only single cell was registered for section-segment mapping. Because of this `_3.dat` had single gid.